_access.qmd

```diff
@@ -0,0 +1,15 @@
+<!--
+In this space, provide an open access link to the case study source files.
+This is necessary for users to get access to case study data, at a minimum.
+See the following example from a previous case study (prior to Motivation section): https://www.opencasestudies.org/ocs-bp-co2-emissions/
+-->
+
+To access the GitHub repository for this case study see here: {LINK}
+
+<!--
+If the data isn't in the main github repository, be sure to describe and link to the open source location of the data and how to access it
+-->
+
+<!--
```

Comment:
ultimately yes or. a new bioOCSdata package probably just so they dont get so big - but depending on if we develop that along side (I think that might be best) the package may or may not be ready)